package.json

```diff
@@ -53,6 +53,7 @@
     "develop": "gatsby develop",
     "start": "npm run develop",
     "build": "rm -rf public && rm -rf .cache && gatsby build",
+    "now-build": "gatsby build",
```

Comment:
If there's `build` script, we use that. But we can override it with `now-build`.
In order to make things clean, we can use this. So, you can still keep the current `build` script.